packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Axis as D3Axis } from 'd3-axis';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33517&commitId=86057e5467c963d3232e10f558270850f253895f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.No beak.chromium.png|2306| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-GaugeChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-GaugeChart.Basic.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>